### DISCUSSION TITLE
Missing/404 PRs for some team members

### DISCUSSION THREAD

**Discussion by jamesl1001** [community/community#54935] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hi all,

My team are experiencing an issue where two of us are unable to see the PRs created by the third member. I am the org owner.

They are not showing on the repo's Pull Requests page and when we go directly to the URL, we get a 404.

The third member has shown us screenshots of it working for them.

I've tried checking the permissions, PR page filters, cleared browser cache.

I'm wondering if it might have something to do with the PR outage at the end of last week...

Any ideas welcome!

**Comment by jamesl1001** [comment, marked as answer]
Turns out that account had been flagged for some reason.

When that's the case, we are unable to see:
- Their PRs
- Their comments on PRs (even though the number of comments is visible on the PR page)
- Their GH profile

**Reply by mayowasamsalad** [reply]
please i am presently experiencing the same thing, how was this solved. my team mates can not see my pr, but they can see the updates i made

**Reply by GBIUNC1atr** [reply]
Hi. Have you received an answer regarding this yet? We are experiencing the same issues and not sure how to fix it. Many thanks.

**Reply by DrXPrime** [reply]
Yes, is there anything that shows on the other person's profile that indicates they need to contact GitHub? How do they resolve the issue on their account. I'm seeing this behavior with another collaborator of mine, and it's very confusing. They can commit (which I see), create issues and reply to issues, but I don't see any of their comments or replies to issues and discussions nor issues they've created. In some cases there's hints that there's a number of replies, but then the issues API only returns my issues.

**Reply by dida0982** [reply]
A conta provavelmente foi sinalizada/restricted pelo GitHub.
Quando isso acontece, outras pessoas podem não conseguir ver:

* Pull Requests da conta
* Comentários em PRs e Issues
* Perfil da conta
* Discussões e respostas

Mesmo assim, os commits ainda podem aparecer normalmente.

Para resolver, a pessoa da conta afetada precisa entrar em contato com o suporte do GitHub:

[[GitHub Support](https://support.github.com/contact?utm_source=chatgpt.com)](https://support.github.com/contact?utm_source=chatgpt.com)

Também vale verificar:

* se recebeu e-mail do GitHub
* se a conta foi marcada como spam/fake
* se falta verificar e-mail ou autenticação
* se houve alguma violação automática detectada pelo sistema

Normalmente só o suporte do GitHub consegue remover esse bloqueio.

**Comment by Chidinma5290** [comment]
I cannot also access the task git hub link to validate PR
